src/manager.js

```diff
@@ -1,4 +1,5 @@
 require('./dfp');
+require('./prebid-load.js');
```

Comment:
We should try to think through a way where this is conditionally required, so we don't need to pull that extra ~78kb of js in until we need it.  Acknowledge that might be difficult, but maybe we put the prebid-load script in this repo but make it the responsibility of the integrating application to bundle them or not?

Reply:
Yea I think doing it on a per app basis would work. We can also do the gpt thing and have prebid-load.js check if it's enabled and append a script tag to head pointed at a hosted prebid.js.

Reply:
Not crazy about that idea, because then you'd have the lag of time for that code to download, execute, and then for prebid to download once you'd append to `<head>`.

Thinking would rather treat Prebid as a separate thing for now, still keep it in this repo, but then have each app decide how it wants to bundle and serve.